MobiusCore/Source/MobiusController.swift

```diff
@@ -33,7 +33,7 @@ public class MobiusController<Model, Event, Effect> {
     private var modelToStartFrom: Model
 
     /// A Boolean indicating whether the MobiusLoop is running or not.
-    public var isRunning: Bool {
+    public var running: Bool {
```

Comment:
These two changes are made in a backwards-incompatible way instead of deprecating the old version and adding a new, as has been done before. I'm fine with that, I think, but I wanted to note that that's a difference in approach. I'm fine with it because I think we're going to make a number of backwards-incompatible changes for 1.0.

Reply:
Look again. :-)

Reply:
Clever trick to fool your reviewers with! :)